### PR TITLE
Modular Production KIT 24.05

### PR DESCRIPTION
**Pull request by dominikoeh**
## Description

- Extension of Shop Floor Information Service with Production Tracking
- BoM semantic model
- Changed banner image file location and converted .png image to .svg
- Semantic models including dependencies

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

@wehrstedt-jan @maximilianong @danielmiehle

### CODE REVIEW

File: docs-kits/kits/Modular Production Kit/page_adoption-view.md [hunk 1]
```diff
@@ -15,74 +15,164 @@ sidebar_position: 2
...
+
+The semantic model can be found here:
+
+- [io.catenax.shopfloor_information.get_production_tracking](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.shopfloor_information.get_production_tracking)
+- [io.catenax.shopfloor_information.provide_production_tracking](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.shopfloor_information.provide_production_tracking)
+
+The *GetProductionTrackingData* model must be implemented by all participants who wish to use the production tracing aspect of the Shop-Floor-Information-Service as a Modular Production, a customer or a participating third party. Companies, which use the Shop-Floor-Information-Service as a customer or third party must be able to send *GetProductionTrackingData* request, whereas partners using the Shop-Floor-Information-Service as a Modular Production must be able to receive the *GetProductionTrackingData* request.
+
+![SIS.Tracking-Parameter](res/SIS.Tracking-Parameter.svg)
```
Comment (HFocken) [inline docs-kits/kits/Modular Production Kit/page_adoption-view.md:83]: This image is not displayed in the rich cliff - please check for all pictures below, as many of them don't seem to be rendered from the link you provided.

**Review by HFocken — CHANGES_REQUESTED** [review]
Thanks for the great KIT - minor change requests - otherwise good to go!

File: docs-kits/kits/Modular Production Kit/page_adoption-view.md [hunk 2]
```diff
@@ -15,74 +15,164 @@ sidebar_position: 2
...
+
+All partners need the same semantic models for a common understanding of the information exchange. Customers or third parties specify within the *GetProductionTrackingData* model the required information about a manufacturing step. This requires that both Modular Production as well as the customer have a common understanding about the product to be manufactured. This common knowledge is referenced for the request.  Modular Production interprets this request and replies with the *ProvideProductionTrackingData* model. Depending on the customer’s request, Modular Production sends the requested information.
+
+The *GetProductionTrackingData* is sent from a partner to Modular Production with a HTTP request to receive production tracking information. The *ProvideProductionTrackingData* model is containing the relevant information like *processParameterName*, *processParameterValue* and *processParameterQuality*
+
+The semantic model can be found here:
```
[the remark this answers is not in the document]
Comment (dominikoeh) [reply docs-kits/kits/Modular Production Kit/page_adoption-view.md:76]: There are standards available in the standard library, yet those will be deprecated by a new standard which is due to release in the current release. I added a link to the currently available standards in the KIT page which contains a link to the standard library

File: docs-kits/kits/Modular Production Kit/page_adoption-view.md (hunk 1 above)
Comment (dominikoeh) [reply docs-kits/kits/Modular Production Kit/page_adoption-view.md]: images appear to render with docusaurus - can you elaborate in which case the images do not render correctly?

Bildschirmfoto 2024-05-23 um 14 09 53

Comment (HFocken) [reply docs-kits/kits/Modular Production Kit/page_adoption-view.md]: In this case, it works fine! Thanks for building it to verify!

**Review by HFocken — APPROVED** [review]
Thanks for the clarifying comments! Good to go!

**Review by danielmiehle — APPROVED** [review]
LGTM

**Review by danielmiehle — APPROVED** [review]

**Review by maximilianong — APPROVED** [review]

**Pull request merged**